apps/src/eval/eval.js

```diff
@@ -106,6 +106,27 @@ Eval.init = function(config) {
       var background = document.getElementById('background');
```

Comment:
Did this, and in doing so realized that the studio version would actually duplicate these when you hit reset. Fixed that.